resources/lang/de_DE/validation.po

```diff
@@ -0,0 +1,36 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: Engelsystem 2.0\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Last-Translator: \n"
+"Language: de_DE\n"
+
+msgid "auth.not-found"
```

Comment:
Is not really a "validation" module output so imho this should be splittet further? In that case it would be something like `auth.po` (or something more general like `pages.po`? but that would mean that we have a ton of files in the end which could be problematic? Eventually we should have a look at the performance impact of that)

Reply:
According to [this](https://www.php.net/manual/de/book.gettext.php#115370) `*.mo` files are cached, so in a production environment the performance impact of having multiple translation files is probably neglectable. In my opinion, though, two files should be sufficient: a `default.po` and (as a more general alternative to naming it `validation.po`) an `errors.po`.

Reply:
If we would use phps own `gettext` that would be true but we are using the [gettext/gettext](https://github.com/php-gettext/Gettext/) Package which allows to differentiate between "No value translated" and "Translated value is the same".

I think that even `errors.po` won't be the right wording as validation messages could be also positive feedback

Reply:
I mistook the `Engelsystem\Helpers\Translation\GettextTranslator` for the `Gettext\GettextTranslator`, the latter of which uses PHP's `gettext`. My bad.

Still, I need a name for the auto-gen messages file. Would a general name like `additional.po` or `other.po` be a solution?

Reply:
At least `additional.po` would be much clearer ;)